Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +9,35 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+
+            var flag = true;  
+            var temp = 0 ;
+            var numLength = 0; 
+
+            if (array == null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887669)